app/common/lib/historyUtil.js

```diff
@@ -144,6 +145,19 @@ const getKey = (siteDetail) => {
   return null
 }
 
+const isHistorySite = (site) => {
+  if (!site) {
+    return false
+  }
+  const order = site.get('order')
+  const location = site.get('location')
+  const tags = site.get('tags')
```

Comment:
for syncing history sites, we actually don't need tags nor type. I thought it might be useful to have a reusable `isHistorySite()` but I think I'll just re-remove it from this PR.